src/main/scala/it/pagopa/interop/apigateway/service/impl/AttributeRegistryProcessServiceImpl.scala

```diff
@@ -53,22 +53,22 @@ class AttributeRegistryProcessServiceImpl(invoker: AttributeRegistryProcessInvok
       }
   } yield result
 
-  override def createAttribute(
-    attributeSeed: AttributeSeed
+  override def createCertifiedAttribute(
+    attributeSeed: CertifiedAttributeSeed
   )(implicit contexts: Seq[(String, String)]): Future[Attribute] = for {
     (bearerToken, correlationId, ip) <- extractHeaders(contexts).toFuture
-    request: ApiRequest[Attribute] = api.createAttribute(
+    request: ApiRequest[Attribute] = api.createCertifiedAttribute(
       xCorrelationId = correlationId,
-      attributeSeed = attributeSeed,
+      certifiedAttributeSeed = attributeSeed,
       xForwardedFor = ip
     )(BearerToken(bearerToken))
     result <- invoker
-      .invoke(request, s"Creating ${attributeSeed.kind} attribute ${attributeSeed.name}")
+      .invoke(request, s"Creating certified attribute ${attributeSeed.name}")
       .recoverWith {
         case err: ApiError[_] if err.code == 409 =>
           (attributeSeed.origin, attributeSeed.code) match {
-            case (Some(origin), Some(code)) => Future.failed(AttributeAlreadyExists(origin, code))
-            case _                          => Future.failed(err)
+            case (origin, code) => Future.failed(AttributeAlreadyExists(origin, code))
+            case _              => Future.failed(err)
```

Comment:
the match is no more required because `origin` and `code` now are not `Option`s.

```suggestion
       Future.failed(AttributeAlreadyExists(origin, code))
```

Here let's add the catch of the code `403` mapping it to `OrganizationIsNotACertifier`